modules/hcp-ibmz-update-reg-ca.adoc

```diff
@@ -0,0 +1,65 @@
+:_mod-docs-content-type: CONCEPT
+[id="hcp-ibmz-update-registry-ca-private-registry_{context}"]
+= Update the registry certificate authority in the AgentServiceConfig resource with the mirror registry
+
+When you use a mirror registry for images, agents need to trust the registry's certificate to securely pull images. Add the certificate authority of mirror registry to the `AgentServiceConfig` custom resource by creating a `ConfigMap`.
+
+.Prerequisites
+
+* You must have installed {mce}.
+
+.Procedure 
+
+. In the same namespace where you installed {mce-short}, create a config map with the mirror registry details. This config map ensures that you grant the hosted cluster workers the capability to retrieve images from the mirror registry.
++
+.Example ConfigMap file
+[source,terminal]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: mirror-config
+  namespace: multicluster-engine
+  labels:
+    app: assisted-service
+data:
+  ca-bundle.crt: |
+    -----BEGIN CERTIFICATE-----
+    -----END CERTIFICATE-----
+  registries.conf: |
+
+    [[registry]]
+      location = "registry.stage.redhat.io"
+      insecure = false
+      blocked = false
+      mirror-by-digest-only = true
+      prefix = ""
+
+      [[registry.mirror]]
+        location = "<mirror_registry>"
+        insecure = false
+
+    [[registry]]
+      location = "registry.redhat.io/multicluster-engine"
+      insecure = false
+      blocked = false
+      mirror-by-digest-only = true
+      prefix = ""
+
+      [[registry.mirror]]
+        location = "<mirror_registry>/multicluster-engine" // <1>
+        insecure = false
+----
++
+<1> Provide the name of the mirror registry.
+
+.  Patch the `AgentServiceConfig` resource to include the config map that you created. If the `AgentServiceConfig` resource is not present, create the `AgentServiceConfig` resource with the following content embedded into it:
```

Comment:
```suggestion
.  Patch the `AgentServiceConfig` resource to include the `ConfigMap` resource that you created. If the `AgentServiceConfig` resource is not present, create the `AgentServiceConfig` resource with the following content embedded into it:
```

Reply:
You might include an optional step here that shows how to create the `AgentServiceConfig` resource.